extensions/ql-vscode/test/vscode-tests/minimal-workspace/local-queries/local-databases.test.ts

```diff
@@ -626,12 +626,15 @@ describe("local databases", () => {
         showNeverAskAgainDialogSpy = jest
           .spyOn(dialog, "showNeverAskAgainDialog")
           .mockResolvedValue("No, and never ask me again");
-        const updateValueSpy = jest.spyOn(Setting.prototype, "updateValue");
+        const setAutogenerateQlPacksSpy = jest.spyOn(
+          config,
+          "setAutogenerateQlPacks",
+        );
 
         await (databaseManager as any).createSkeletonPacks(mockDbItem);
 
         expect(generateSpy).not.toBeCalled();
-        expect(updateValueSpy).toHaveBeenCalledWith("never", 2);
+        expect(setAutogenerateQlPacksSpy).toHaveBeenCalledWith("never", 1);
```

Comment:
I'm surprised that this used to pass with `2` 🤔 Just stepping through the code, I can't find more than one place where we update the value of a setting 😅

Reply:
Oh nevermind, this isn't about the number of calls. `updateValue` expected 2 arguments, but `setAutogenerateQlPacks` only one 🤦🏽